bank-account-test/src/main/kotlin/br/com/challenges/Account.kt

```diff
@@ -0,0 +1,36 @@
+package br.com.challenges
+
+class Account(
+    val numberAccount: Int,
+    val nameClient: String,
+    accountBalance: Double,
+) {
```

Comment:
Para validar o numberAccount deveriamos ter um método contructor recebendo os argumentos e na sequência fazer a validação destes if (numberAccount > 1002) { aqui poderiamos apresentar a exceção } else { this.numberAccount = numberAccount

Reply:
No próximo papo, retomamos esse assunto. Nesse momento podemos fechar esse PR :)